src/main/java/ca/corefacility/bioinformatics/irida/model/workflow/submission/AnalysisSubmission.java

```diff
@@ -651,70 +406,49 @@ public static Builder builder(UUID workflowId) {
 
 	/**
 	 * Whether or not a remoteAnalysisId exists for this submission.
-	 * 
-	 * @return True if a remoteAnalysisId exists for this submission, false
-	 *         otherwise.
+	 *
+	 * @return True if a remoteAnalysisId exists for this submission, false otherwise.
 	 */
 	public boolean hasRemoteAnalysisId() {
 		return remoteAnalysisId != null;
 	}
 
 	/**
 	 * Whether or not a remoteWorkflowId exists for this submission.
-	 * 
-	 * @return True if a remoteWorkflowId exists for this submission, false
-	 *         otherwise.
+	 *
+	 * @return True if a remoteWorkflowId exists for this submission, false otherwise.
 	 */
 	public boolean hasRemoteWorkflowId() {
 		return remoteWorkflowId != null;
 	}
 
 	/**
 	 * Whether or not a remoteInputDataId exists for this submission.
-	 * 
-	 * @return True if a remoteInputDataId exists for this submission, false
-	 *         otherwise.
+	 *
+	 * @return True if a remoteInputDataId exists for this submission, false otherwise.
 	 */
 	public boolean hasRemoteInputDataId() {
 		return remoteInputDataId != null;
 	}
-	
-	/**
-	 * Sets flag to indicate whether or not samples in the submission should be updated with analysis results following completion.
-	 * @param updateSamples If true, updates samples from results on completion.
-	 */
-	public void setUpdateSamples(boolean updateSamples) {
-		this.updateSamples = updateSamples;
-	}
-
-	/**
-	 * Whether or not to update samples from results on completion.
-	 * @return Update samples from results on completion.
-	 */
-	public boolean getUpdateSamples() {
-		return updateSamples;
-	}
 
 	@Override
 	public int hashCode() {
 		return Objects.hash(name, workflowId, remoteAnalysisId, remoteInputDataId, remoteWorkflowId, createdDate,
-				modifiedDate, analysisState, analysisCleanedState, analysis, referenceFile, namedParameters, submitter, priority);
+				modifiedDate, analysisState, analysisCleanedState, analysis, referenceFile, namedParameters, submitter,
+				priority);
 	}
 
 	@Override
 	public boolean equals(Object other) {
 		if (other instanceof AnalysisSubmission) {
 			AnalysisSubmission p = (AnalysisSubmission) other;
 			return Objects.equals(createdDate, p.createdDate) && Objects.equals(modifiedDate, p.modifiedDate)
-					&& Objects.equals(name, p.name) && Objects.equals(workflowId, p.workflowId)
-					&& Objects.equals(remoteAnalysisId, p.remoteAnalysisId)
-					&& Objects.equals(remoteInputDataId, p.remoteInputDataId)
-					&& Objects.equals(remoteWorkflowId, p.remoteWorkflowId)
-					&& Objects.equals(analysisState, p.analysisState)
-					&& Objects.equals(analysisCleanedState, p.analysisCleanedState)
-					&& Objects.equals(referenceFile, p.referenceFile) && Objects
-					.equals(namedParameters, p.namedParameters) && Objects.equals(submitter, p.submitter) && Objects
-					.equals(priority, p.priority);
+					&& Objects.equals(name, p.name) && Objects.equals(workflowId, p.workflowId) && Objects.equals(
```

Comment:
It looks like you're missing `analysis` from the `equals()` method, but it is present in `hashCode()`. These should be made consistent.

Reply:
Yeah looks like that was missing back in dev too.  Added in fd05858.  Thanks for catching it.